.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -201,10 +201,15 @@ jobs:
     displayName: "Install oras and build the payload json file"
     condition: eq(variables.IS_PR, false)
 
-  - task: EsrpCodeSigning@3
+  - task: EsrpCodeSigning@5
     condition: eq(variables.IS_PR, false)
     inputs:
-      ConnectedServiceName: 'AME ESRPContainerInsights Service Connection'
+      ConnectedServiceName: 'AME ESRPContainerInsights MSI Service Connection'
+      AppRegistrationClientId: 'bc3b3065-9256-4473-a288-e419f37f4d6d'
+      AppRegistrationTenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47'
```

Comment:
@wanlonghenry , can we make these pipeline variables and pass from the pipelines instead of exposing in public GitHub code?

Reply:
changed